extensions/resource-deployment/src/ui/eulaWizard/eulaSettingsPage.ts

```diff
@@ -0,0 +1,576 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as nls from 'vscode-nls';
+import { EulaWizard } from './eulaWizard';
+import { AgreementInfo, DeploymentProvider, ITool, ResourceType, ResourceTypeOptionValue, ToolRequirementInfo, ToolStatus } from '../../interfaces';
+import { EulaWizardPage } from './eulaWizardPage';
+import { createFlexContainer } from '../modelViewUtils';
+import { getErrorMessage } from '../../utils';
+import { EOL } from 'os';
+import * as loc from '../../localizedConstants';
+
+
+const localize = nls.loadMessageBundle();
```

Comment:
All localized constants should go in https://github.com/microsoft/azuredatastudio/blob/main/extensions/resource-deployment/src/localizedConstants.ts